algorithms/src/Kokkos_Random.hpp

```diff
@@ -661,14 +661,17 @@ class Random_XorShift64 {
   // number
   KOKKOS_INLINE_FUNCTION
   double normal() {
+#ifndef KOKKOS_ENABLE_HIP
```

Comment:
Can we mark this with something like //KOKKOS_IMPL_HIP_WORKAROUND

Reply:
Maybe better use the IMPL_ macro directly then

Reply:
Maybe, I just want us to make the workarounds easily grep able, and sortable by HIP vs OpenMPTarget etc.